src/custom/components/Link/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { ExternalLink } from 'theme/index'
+import HashLink from 'components/HashLink'
+
+export { ExternalLink } from 'theme/index'
+
+export interface LinkRendererProps {
+  href: string
+  children: React.ReactNode
+  smooth?: boolean
+  scroll?: ((element: HTMLElement) => void) | undefined
+}
+
+export function Link(props: LinkRendererProps) {
+  const { children, href = '#', smooth, scroll } = props
+  const isExternalLink = /^(https?:)?\/\//.test(href)
+  return isExternalLink ? (
+    <ExternalLink href={href}>{children}</ExternalLink>
+  ) : (
+    <HashLink smooth={smooth} to={href} scroll={scroll}>
+      {children}
+    </HashLink>
+  )
+}
+
+export function LinkScrollable(props: LinkRendererProps): JSX.Element {
+  const { children, smooth = true, ...otherProps } = props
+
+  const scrollWithOffset = (el: HTMLElement) => {
+    const yCoordinate = el.getBoundingClientRect().top + window.pageYOffset
+    const yOffset = -24
```

Comment:
:sparkles:

Reply:
this mean magic number :D

Reply:
Why I get the blame  of some code i didn't even do 😂. I don't even know what's that :) 

This is just a refactor of some code that someone in the team did some time ago. Not giving names, only mentioning that he is blonde

Reply:
Let's review this, but for DEVELOP. Okok?